omero/sysadmins/dropbox.txt

```diff
@@ -41,13 +41,16 @@ the following more specific requirements:
 
 .. note:: It is likely, but not guaranteed, that DropBox will run on later
     versions of Windows, however only tested versions are supported by default.
-    Setting the property :property:`omero.fs.platformCheck` to ``False`` in
+    
+    Setting the property :property:`omero.fs.platformCheck` to ``False``
+    in,
+    
```

Comment:
Not sure what all this extra whitespace is for, the rendered box looks rather odd with these added linebreaks given it is all written as one sentence